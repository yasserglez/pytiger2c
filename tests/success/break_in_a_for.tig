for i := 1 to 10 do
    if i <= 5 then
        print("This should be printed 5 times.\n")
    else
        break
