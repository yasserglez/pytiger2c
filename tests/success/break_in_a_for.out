This should be printed 5 times.
This should be printed 5 times.
This should be printed 5 times.
This should be printed 5 times.
This should be printed 5 times.
